### PR TITLE
fix eslint issue

### PR DESCRIPTION
**Pull request by jannisvisser**

### CODE REVIEW

File: services/121-service/package.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
   "homepage": "https://www.121.global/",
   "main": "index.js",
   "scripts": {
-    "lint": "npx eslint '**/*.ts'",
+    "lint": "npx eslint **/*.ts",
```
Comment (elwinschmitz) [inline services/121-service/package.json:8]: Not like this?
```suggestion
    "lint": "eslint '**/*.ts'",
```

File: services/PA-accounts-service/package.json [hunk 2]
```diff
@@ -5,7 +5,7 @@
     "homepage": "https://www.121.global/",
     "main": "index.js",
     "scripts": {
-        "lint": "npx eslint '**/*.ts'",
+        "lint": "npx eslint **/*.ts",
```
Comment (elwinschmitz) [inline services/PA-accounts-service/package.json:8]:
```suggestion
        "lint": "eslint '**/*.ts'",
```

File: services/121-service/package.json (hunk 1 above)
Comment (jannisvisser) [reply services/121-service/package.json:8]: no that doesn't work for me..

Comment (jannisvisser) [reply services/121-service/package.json:8]: @elwinschmitz

**Pull request merged**